wdqs-proxy/latest/entrypoint.sh

```diff
@@ -1,11 +1,15 @@
-#!/bin/sh
+#!/usr/bin/env bash
```

Comment:
I don't think this image has bash

Reply:
Great, does that mean we have to keep two versions of the variable checking logic, using sh in some images and bash in others?

Reply:
Probably :(

I originally tried to write it all in a portable way (hence why they all looked a little ugly and used things like `${REQUIRED_VARIABLES[@]}`).

I guess there was a point at which those with bash available used some bash only features. If it's not that excessive then perhaps we should try to make them all use /bin/sh rather than bash.